tasks/office.py

```diff
@@ -163,9 +159,61 @@ def get_moves_queue(self, overrides=None):
             **self.user.cert_kwargs,
         )
         moves, success = check_response(resp, "getServicesCounselingQueue")
+        if success:
+            # these are not full move models and will be those in needs counseling status
+            self.add_stored(MOVE_TASK_ORDER, moves["queueMoves"])
+
+    @tag(ORDER, "getOrder")
+    @task
+    def get_orders(self, overrides=None):
+        """
+        Fetches a single service member's orders
+        """
+        # If id was provided, get that specific one. Else get any stored one.
+        object_id = overrides.get("id") if overrides else None
+        move = self.get_stored(MOVE_TASK_ORDER, object_id)
+        if move is None:
+            logger.info("Skipping get orders no move is stored yet")
+            return
+
+        orders_id = move.get("ordersId")
+        if orders_id is None:
+            move = self.get_move({"id": move["id"]})
```

Comment:
It feels weird to see a dict passed in like this instead of a keyword argument, but I guess it's not wrong? Not really sure where I land on that

Reply:
That was your overrides functionality if I'm understanding it correctly.  Are you saying we don't need it to be so generalized?

Reply:
OH, I was confused, I thought this was some kind of `get_stored` call. But it's an endpoint! Blah, sorry 😄